tests/queryRoot.js

```diff
@@ -0,0 +1,26 @@
+
+define([
+    "esquery",
+    "jstestr/assert",
+    "jstestr/test",
+    "./fixtures/nestedFunctions",
+    "./fixtures/nestedFunctionsWithReturns",
+], function (esquery, assert, test, nestedFunctions, nestedFunctionsWithReturns) {
+
+    test.defineSuite("root selector", {
+
+        "select only first level function": function () {
+            var matches = esquery(nestedFunctions, ":root > FunctionDeclaration");
+            assert.isSame(1, matches.length);
+            assert.isSame(matches[0].id.name, "foo")
+        },
+
+        "select only first level return": function () {
+            var firstBlock =  esquery(nestedFunctionsWithReturns, "BlockStatement")[0];
+            var matches = esquery(firstBlock, ":scope > ReturnStatement");
```

Comment:
I don't really understand how this tests something different than if it was `:root > ReturnStatement`, could you please explain it to me?

Reply:
Or is it actually a typo, since the next file does the same test?

Reply:
It was a typo ... 😓